test/unit/org/apache/cassandra/service/ClientStateTest.java

```diff
@@ -22,18 +22,12 @@
 import java.util.concurrent.atomic.AtomicInteger;
 
 import com.google.common.collect.Iterables;
+import org.apache.cassandra.auth.*;
```

Comment:
nit: Probably want to at least move this back down w/ the other C* imports, if not move and get rid of the wildcard as well.